src/LookupModel.php

```diff
@@ -57,7 +57,7 @@ public function __construct(array $lookup)
         $this->latitude = $this->getOrNull('latitude', $lookup);
         $this->longitude = $this->getOrNull('longitude', $lookup);
         $timezone = $this->getOrNull('timezone', $lookup);
-        if (!is_null($timezone)) {
+        if (!is_null($timezone) && in_array($timezone, timezone_identifiers_list())) {
```

Comment:
Honestly, it makes me uneasy to presume that there will only ever be 2 invalid responses. While I do like the more simple `if ($timezone !== 'n/a')` if a timezone falls through the cracks or just is invalid it should be validated against more thoroughly than just the simple `!==` statement. I'd err on the side of caution here and go with the `in_array` check.